lib/ui/home/tabs/locker_tab/apps_item.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:cobble/domain/db/models/app.dart';
+import 'package:cobble/domain/apps/app_manager.dart';
+import 'package:cobble/domain/entities/hardware_platform.dart';
+import 'package:cobble/ui/common/components/cobble_button.dart';
+import 'package:cobble/ui/common/components/cobble_tile.dart';
+import 'package:cobble/ui/common/icons/fonts/rebble_icons.dart';
+import 'package:cobble/ui/home/tabs/locker_tab/apps_sheet.dart';
+import 'package:cobble/ui/theme/with_cobble_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg_provider/flutter_svg_provider.dart';
+
+class AppsItem extends StatelessWidget {
+  final App app;
+  final bool compatible;
+  final AppManager appManager;
+  final PebbleWatchLine? lineConnected;
+  final int? index;
+
+  const AppsItem(
```

Comment:
Let's follow Flutter's design and make all Widget arguments named. You can use `required` keyword when they are required.